### PR TITLE
config(amazonq): update Q API model

### PR DESCRIPTION
**Pull request by Will-ShaoHua**
## Problem
requested by partner team

## Solution

---

License: I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/core/src/codewhisperer/client/user-service-2.json [hunk 1]
```diff
@@ -2,11 +2,14 @@
...
     "metadata": {
         "apiVersion": "2022-11-11",
+        "auth": ["smithy.api#httpBearerAuth"],
         "endpointPrefix": "amazoncodewhispererservice",
         "jsonVersion": "1.0",
         "protocol": "json",
+        "protocols": ["json"],
         "serviceFullName": "Amazon CodeWhisperer",
         "serviceId": "CodeWhispererRuntime",
+        "signatureVersion": "bearer",
```
Comment (Will-ShaoHua) [inline packages/core/src/codewhisperer/client/user-service-2.json:12]: not sure do we need to include these 3 lines ornot

Comment (Will-ShaoHua) [reply packages/core/src/codewhisperer/client/user-service-2.json:12]: think it should be fine, confirmed jb has these. https://github.com/aws/aws-toolkit-jetbrains/blob/main/plugins/core/sdk-codegen/codegen-resources/codewhispererruntime/service-2.json#L4-L15

**Review by justinmk3 — APPROVED** [review]

**Comment by Will-ShaoHua** [comment]
@justinmk3 fyi i will also request our service team to review the PR then merge

**Review by fangwade007 — APPROVED** [review]

File: packages/core/src/codewhisperer/client/user-service-2.json (hunk 1 above)
Comment (dhasani23) [reply packages/core/src/codewhisperer/client/user-service-2.json:12]: Should not include

**Pull request merged**